### PR TITLE
docs: add clarifications to the order of list response in query

### PR DESCRIPTION
**Pull request by 170210**
## Description
This PR adds clarifications to the order of list response in x/wasm query, which will make it easier for developers to understand the returned order.
closes: #91 

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.(not need)
- [ ] I have updated the documentation accordingly.(not need)
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`(not need)

### CODE REVIEW

File: docs/proto/proto-docs.md [hunk 1]
```diff
@@ -1081,7 +1081,7 @@ Query/ContractHistory RPC method
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| `entries` | [ContractCodeHistoryEntry](#cosmwasm.wasm.v1.ContractCodeHistoryEntry) | repeated |  |
+| `entries` | [ContractCodeHistoryEntry](#cosmwasm.wasm.v1.ContractCodeHistoryEntry) | repeated | return in the order of contract execution |
```
Comment (loloicci) [inline docs/proto/proto-docs.md:1084]: "contract execution" is ambiguous. Does this mean "contract updated timestamp"?

File: docs/proto/proto-docs.md [hunk 2]
```diff
@@ -1148,7 +1148,7 @@ Query/ContractsByCode RPC method
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| `contracts` | [string](#string) | repeated | contracts are a set of contract addresses |
+| `contracts` | [string](#string) | repeated | contracts are a set of contract addresses return in the order of instantiation |
```
Comment (loloicci) [inline docs/proto/proto-docs.md:1151]: need a break line or period before "return".
And, isn't it the order of "instantiation or migration"?

Comment (170210) [reply docs/proto/proto-docs.md:1151]: > need a break line or period before "return".

I attempted to create a line break using a `<br>` tag, which resolved the issue here. However, in [query.pb.go](https://github.com/Finschia/wasmd/pull/103/files#diff-c5afb1f23df96451f326680e605bd104db59c63e88c5480a634f61610e627334)  at line 273, it became 
`// contracts are a set of contract addresses<br>`

So I will try to make a period before

File: proto/cosmwasm/wasm/v1/query.proto [hunk 3]
```diff
@@ -97,7 +97,7 @@ message QueryContractHistoryRequest {
 // QueryContractHistoryResponse is the response type for the
 // Query/ContractHistory RPC method
 message QueryContractHistoryResponse {
-  // return in the order of contract execution
+  // return in the order of timestamps according to contract updates
```
Comment (loloicci) [inline proto/cosmwasm/wasm/v1/query.proto:100]: updates -> updated

**Review by loloicci — APPROVED** [review]

**Review by da1suk8 — APPROVED** [review]

**Pull request merged**